pkg/admission/cluster.go

```diff
@@ -23,6 +23,11 @@ type clusterValidator struct {
 }
 
 func (c *clusterValidator) Admit(response *webhook.Response, request *webhook.Request) error {
+	if request.Operation == admissionv1.Delete {
```

Comment:
This webhook should only be called for create/update.  Can we just define a second ValidatingWebhook instead of combining the two?

Reply:
Separated the two and left cluster as-is.